From a3f30060e509df68b940a4e16154f6c003bf073b Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Fri, 8 Mar 2013 17:16:20 +0800
Subject: [PATCH 153/225] ENGR00243315-6 ADV7180:Remove unnecessary header
 files

This patch removes unnesessary header files in the driver.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 871c22325d619ee1c4d2107ab9298e7ef5193c21)
---
 drivers/media/video/mxc/capture/adv7180.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/media/video/mxc/capture/adv7180.c b/drivers/media/video/mxc/capture/adv7180.c
index 9eb18ca..3c12fdc 100644
--- a/drivers/media/video/mxc/capture/adv7180.c
+++ b/drivers/media/video/mxc/capture/adv7180.c
@@ -21,16 +21,11 @@
 
 #include <linux/module.h>
 #include <linux/init.h>
-#include <linux/slab.h>
-#include <linux/ctype.h>
 #include <linux/types.h>
 #include <linux/delay.h>
-#include <linux/semaphore.h>
 #include <linux/device.h>
 #include <linux/i2c.h>
-#include <linux/wait.h>
 #include <linux/videodev2.h>
-#include <linux/workqueue.h>
 #include <linux/regulator/consumer.h>
 #include <linux/fsl_devices.h>
 #include <media/v4l2-chip-ident.h>
-- 
1.7.10.4

